hw/xbox/mcpx_apu.c

```diff
@@ -55,7 +55,9 @@
 #define NV_PAPU_XGSCNT                                   0x0000200C
```

Comment:
Can we move register definitions to a mpcx_apu_regs.h? I'm ok for follow up PR doing so.

Reply:
Yeah, I also want to see this, especially with a lot of VP stuff coming in.
But it should be done in a follow-up.

*(Ideally someone else will do it)*